public/components/common/form/input_text.tsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import { EuiFieldText } from '@elastic/eui';
-import { IInputFormType } from "./types";
+import { IInputFormType } from './types';
 
-export const InputFormText = ({ value, isInvalid, onChange }: IInputFormType) => {
-	return (
-		<EuiFieldText
-			fullWidth
-			value={value}
-			isInvalid={isInvalid}
-			onChange={onChange}
-		/>
-	);
+export const InputFormText = ({
+  value,
+  isInvalid,
+  onChange,
+  placeholder,
+  fullWidth,
+}: IInputFormType) => {
+  return (
+    <EuiFieldText
+      fullWidth={typeof 'fullWidth' === 'undefined' ? true : fullWidth}
```

Comment:
**nitpick**: the typeof has a problem.

**sugesstion**: remove the single quotes around the `fullWidth` if you want to use the `fullWidth` variable.

```ts
typeof fullWidth === 'undefined' ? true : fullWidth

```